compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ImageComposeScene.skiko.kt

```diff
@@ -55,6 +55,11 @@ import org.jetbrains.skiko.currentNanoTime
 /**
  * Render Compose [content] into an [Image]
  *
+ * This function doesn't support concurrent use from different threads. This includes use from a
```

Comment:
It should be supported if we fix the `sendApplyNotifications` issue. Can you create and link with the Google tracker issue for the `Recomposer`?

Reply:
I will create an issue, but this warning is not about that.

The warning refers to the problem caused by 
- https://issuetracker.google.com/issues/283162626
- https://issuetracker.google.com/issues/283216580
- Possibly others too.